### ISSUE TITLE
Late callouts for Stigma-4 and EX2

### ISSUE THREAD

**Issue by YehnBeep** [quisquous/cactbot#3880]
Hi,

I've noticed callouts can be a bit late for the in/out mechanics of the two bosses in the title, making it difficult to dodge if one is as of yet unfamiliar with the mechanics. In the case of Stigma-4, it is particularly problematic because the callout activates at the same time as Mindjack stuns the player, which would render newer players unable to react at all.

In the case of the second extreme trial, this might be more personal, but I tend to prefer to rely on aural cues - however, the cactbot callout is so late that an aural cue tied to it ends up being useless and I must rely on the visual cue to react in time.

Would it be possible to improve the response time of these callouts to make them more useful?

Thank you.

**Comment by JLGarber** [comment]
For both encounters, is this a consistent thing and definitely not related to #3824? I was actually just running Weapon EX myself tonight and didn't notice anything particularly out of place.

As for Dreamscape, I'll take a look next time I'm in there, but in testing things I didn't notice it being *that* difficult to adjust for Mindhack. It's possible we want to call it a second earlier.

**Comment by YehnBeep** [comment]
Hi,

I can't be sure it's not related to #3824, but I don't think I had ACT running for that long. Other callouts seemed to be on point. Next time I'm in one of the fights, I'll try to keep an eye on my task manager. I've used cactbot in previous expansions and in previous ex fights, such as weapon, I haven't noticed the issue before, and that's why this seemed odd to me.

For EX2, it was consistently behind the visual telegraph for me - I gave up on trying to use cactbot's aural cue.

For Stigma-4... I typically only see one mindjack + in/out in a fight and haven't had it come up since, so I'm afraid I'm not sure on that one.
